smac/facade/abstract_facade.py

```diff
@@ -290,7 +290,23 @@ def tell(self, info: TrialInfo, value: TrialValue, save: bool = True) -> None:
             Whether the runhistory should be saved.
         """
         return self._optimizer.tell(info, value, save=save)
-
+    
+    def warmstarting(self, infos, values, save: bool = True) -> None:
+        """Adds the result of a list of trials to the runhistory and updates the intensifier.
+
+        Arguments:
+            infos (List TrialInfo): A list of Trial Info which describes the trials from which to process the results.
+            values (List float): A list containing per item the relevant information regarding the execution of a trial.
+            save (bool, optional): Whether the runhistory should be saved. Defaults to True.
+        """
+        # Check length of info is same as Values
+        if(len(infos)!=len(values)):
+            raise ValueError('Not matching lenght, the lenght of Info list must match the values list')
+        trialValues = [TrialValue(cost=cost) for cost in values]
```

Comment:
[TrialValue](https://github.com/automl/SMAC3/blob/main/smac/runhistory/dataclasses.py#L94) also requires some additional arguments (such as time, status, and additional_info). Therefore, it would be nice to check the content of values and then decide how to handle it.
For instance, if `values` is a list of `dict`, then we initialize `TrialValue` with `**value` , or if `values` is a list of `TrialValue`, then we do not need to transform that to `TrialValue`.